TwitchDownloaderCore/TwitchObjects/ChatRoot.cs

```diff
@@ -97,10 +97,24 @@ public class EmbedEmoteData
     public int height { get; set; }
 }
 
+public class EmbedChatBadge
+{
+    public string name { get; set; }
+    public Dictionary<string, byte[]> versions { get; set; }
+}
+
+public class EmbedCheerEmote
+{
+    public string prefix { get; set; }
+    public Dictionary<int, EmbedEmoteData> tierList { get; set; }
+}
+
 public class Emotes
```

Comment:
Maybe create new sections for embeded badges and bits

Reply:
doing that means no dealing with "upgrading" old chat files because the `TryGetProperty` would return false

Reply:
Thought on it a bit more. Keeping the single class could be nice IF we rename it to something more appropriate like "embeddedData". Still we do not need to deal with "upgrading" old jsons because we can `TryGetProperty("embeddedData")` else `TryGetProperty("emotes")`.

We can also change `--embed-emotes` to `--embed-data` and add a simple find and replace to `ConvertFromOldSyntax` for backwards compatibility.